draft-ietf-httpbis-priority.md

```diff
@@ -298,7 +298,7 @@ Servers can send SETTINGS_NO_RFC7540_PRIORITIES with a value of 1 to indicate
 that they will ignore HTTP/2 priority signals sent by clients.
 
 Endpoints that send SETTINGS_NO_RFC7540_PRIORITIES are encouraged to use
-alternative priority signals (for example, see {{header-field}} or
+alternative priority signals (for example, see Sections {{header-field}} or
```

Comment:
the general RFC style seems to be:

Sections X and Y

Section X or Section Y

This make logical sense to me, so I would drop this change.

Reply:
Thank you for the clarification. Reverted in 3ae5fd7.